### PR TITLE
Finish reformatting + simplifying bronze module java code

### PR DESCRIPTION
**Pull request by nathangong**
I wanted to make java code from some bronze modules easier to read by adding Kattio instead of buffered reader and formatting the code to follow Java conventions and USACO Guide conventions

### CODE REVIEW

File: content/2_Bronze/Complete_Rec.mdx [hunk 1]
```diff
@@ -782,7 +783,7 @@ import java.util.*;
 
 public class Chessboard {
 	public static boolean ok[][] = new boolean[8][8];
-	public static ArrayList < Integer > perm = new ArrayList < Integer > ();
+	public static ArrayList<Integer> perm = new ArrayList<Integer>();
```
Comment (dongliu0426) [inline content/2_Bronze/Complete_Rec.mdx:786]: `List<Integer>` https://usaco.guide/general/adding-solution?lang=java#java

**Review by dongliu0426 — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Complete_Rec.mdx (hunk 1 above)
Comment (nathangong) [reply content/2_Bronze/Complete_Rec.mdx]: Fixed

**Review by dongliu0426 — APPROVED** [review]

File: content/2_Bronze/Complete_Rec.mdx [hunk 2]
```diff
@@ -782,7 +783,7 @@ import java.util.*;
 
 public class Chessboard {
 	public static boolean ok[][] = new boolean[8][8];
-	public static ArrayList < Integer > perm = new ArrayList < Integer > ();
+	public static List<Integer> perm = new ArrayList<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Complete_Rec.mdx:786]:
```suggestion
	public static List<Integer> perm = new ArrayList<>();
```

File: content/2_Bronze/Complete_Rec.mdx [hunk 3]
```diff
@@ -643,11 +643,11 @@ import java.util.*;
 
 public class Test {
 	static boolean[] used;
-	static ArrayList<Integer> cur = new ArrayList<Integer>();
+	static List<Integer> cur = new ArrayList<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Complete_Rec.mdx:646]:
```suggestion
	static List<Integer> cur = new ArrayList<>();
```

File: content/2_Bronze/Intro_Complete.mdx [hunk 4]
```diff
@@ -61,23 +61,44 @@ int main() {
...
+		}
+
+		int max = 0; // storing the current maximum
+		for (int i = 0; i < n; i++) { // for each first point
+			for (int j = i + 1; j < n; j++) { // for each second point
+				int dx = x[i] - x[j];
+				int dy = y[i] - y[j];
+				// if the square of the distance between the two points is greater than
+				// our current maximum, then update the maximum
+				max = Math.max(max, dx*dx + dy*dy);
```
Comment (maggie-j-liu) [inline content/2_Bronze/Intro_Complete.mdx:90]:
```suggestion
				max = Math.max(max, dx * dx + dy * dy);
```

File: content/2_Bronze/Intro_Sets.mdx [hunk 5]
```diff
@@ -673,10 +670,11 @@ unlikely to work, as it will probably cause a
 See the following snippet for an example:
 
 ```java
-void iterate_remove_set_BAD() {
+void iterateRemoveBad() {
 	Set<Integer> s = new TreeSet<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Sets.mdx:677]:
```suggestion
 	Set<Integer> s = new TreeSet<>();
```

File: content/2_Bronze/Intro_Sets.mdx [hunk 6]
```diff
@@ -686,21 +684,22 @@ One work-around is to use `Iterator` and the `.remove()` method to remove
 elements while looping over them, like in the next code snippet:
 
 ```java
-void iterate_remove_set() {
+void iterateRemoveSet() {
 	Set<Integer> s = new TreeSet<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Sets.mdx:690]:
```suggestion
 	Set<Integer> s = new TreeSet<>();
```

File: content/2_Bronze/Intro_Sets.mdx [hunk 7]
```diff
@@ -720,21 +719,23 @@ method that you call on your original Collection. See the following code snippet
 for an example (it works equivalently to the code above):
 
 ```java
-void iterate_remove_set_good() {
+void iterateRemoveSetGood() {
 	Set<Integer> s = new TreeSet<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Sets.mdx:724]:
```suggestion
 	Set<Integer> s = new TreeSet<>();
```

File: content/2_Bronze/Intro_Sets.mdx [hunk 8]
```diff
@@ -720,21 +719,23 @@ method that you call on your original Collection. See the following code snippet
...
-	//s starts as {0, 1, 2}
+	// s starts as {0, 1, 2}
 	s.add(0); s.add(1); s.add(2);
 
 	Set<Integer> toRemove = new TreeSet<Integer>();
```
Comment (HarryWangATX) [inline content/2_Bronze/Intro_Sets.mdx:728]:
```suggestion
 	Set<Integer> toRemove = new TreeSet<>();
```

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Intro_Sets.mdx (hunk 5 above)
Comment (nathangong) [reply content/2_Bronze/Intro_Sets.mdx:677]: For some reason this change suggestion is bugged and github won't let me commit it - I'll just do the rest manually

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

**Review by brad182 — APPROVED** [review]

**Comment by nathangong** [comment]
Also oops I just read the discussion about Kattio not being the recommended standard anymore so if anyone thinks that this PR isn't necessary then I can close it and replace it with just some small code cleanups.

File: content/2_Bronze/Intro_Greedy.mdx [hunk 9]
```diff
@@ -107,18 +107,18 @@ int main() {
...
-		PrintWriter pw = new PrintWriter(new BufferedWriter(new FileWriter("breedflip.out")));
-		int n = Integer.parseInt(br.readLine());
-		char[] a = br.readLine().toCharArray();
-		char[] b = br.readLine().toCharArray();
+		Kattio io = new Kattio("breedflip");
+		int n = io.nextInt();
+		char[] a = io.next().toCharArray();
+		char[] b = io.next().toCharArray();
 		int ret = 0;
 		while(!new String(a).equals(new String(b))) {
```
Comment (TruaShamu) [inline content/2_Bronze/Intro_Greedy.mdx:123]: Would adding spacing to the while and for loops make it look better?
```suggestion
		while (!new String(a).equals(new String(b))) {
```

Comment (TruaShamu) [reply content/2_Bronze/Intro_Greedy.mdx:123]: And also on lines 126, 128, 129, and 130 (between the if statement and the parenthesis)

File: content/2_Bronze/Complete_Rec.mdx [hunk 10]
```diff
@@ -902,6 +903,7 @@ public class Main {
 	static String g[];
 	static boolean sum[], dif[], c[];
 	static int ans = 0;
+	
 	static void dfs(int r) { //place queen in row r
```
Comment (TruaShamu) [inline content/2_Bronze/Complete_Rec.mdx:907]: Add spacing at the if statement at line 913, and also at the for loop on line 925 please -

Comment (TruaShamu) [reply content/2_Bronze/Complete_Rec.mdx:907]: (Sorry, this is my first time reviewing, so I'm not very good at it)

File: content/2_Bronze/Intro_Sets.mdx [hunk 11]
```diff
@@ -73,7 +73,7 @@ already present, `remove`, which deletes an element if it exists, and
 `contains`, which checks whether the set contains that element.
```
Comment (TruaShamu) [inline content/2_Bronze/Intro_Sets.mdx:73]: Spacing on line 88, before the opening parenthesis, and then after the closing parenthesis

File: content/2_Bronze/Intro_Greedy.mdx (hunk 9 above)
Comment (nathangong) [reply content/2_Bronze/Intro_Greedy.mdx:123]: It seems that having no spacing there is a common styling pattern for c++ and sometimes java. I agree that having a space does make it look better, @maggie-j-liu should I change it or leave it?

Comment (maggie-j-liu) [reply content/2_Bronze/Intro_Greedy.mdx:123]: Up to you, I prefer having a space but as long as the code is consistent, it's fine.

Comment (nathangong) [reply content/2_Bronze/Intro_Greedy.mdx:123]: Alright I can add spaces to make the code consistent with everything else then

**Review by TruaShamu — APPROVED** [review]

**Review by TruaShamu — APPROVED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**